### DISCUSSION TITLE
there is bug with passoword

### DISCUSSION THREAD

**Discussion by hirockstar** [spencerwooo/onedrive-vercel-index#732] [General]
there is bug with passoword proctacion it can bypass easily with this small changes 
https://drive.swo.moe/%F0%9F%8C%9E%20Private%20folder/u-need-a-password/
https://drive.swo.moe/%F0%9F%8C%9E%20Private%20folder/U-need-a-password/
 you just need to change the url folder name to small or big 
 @spencerwooo @myl7 please look in to and fix it.

**Comment by myl7** [comment]
Fixed in #734

**Reply by hirockstar** [reply]
Bro help me to add sitemap.xml in to it i try everything did not succes yet can you help me to do that please
